tests/test_deploy_github_pages.py

```diff
@@ -369,8 +386,8 @@ def test_abort_local_uncommitted_changes_exist_in_source_branch(setup_test_env):
                                                  "--source_branch", source_branch,
                                                  "--module_path", "../test_package", "../another_test_package"])
     assert e.match(f"Abort, you have uncommitted changes in source branch  {source_branch}, "
-                     f"please commit and push the following files:\n .*")
-    assert e.type == SystemExit
+                     f"please commit and push the following files:\n .*") \
+           and e.type == SystemExit
```

Comment:
You may also wan't to consider making it a `multi line assert` using the `()` syntax.